### PR TITLE
[NUI] Add Padding parameter for TextPageUtil

### PR DESCRIPTION
**Pull request by JoogabYun**
### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/public/Utility/TextPageUtil.cs [hunk 1]
```diff
@@ -27,318 +27,331 @@
...
-    private string pageString;
-
     /// <summary>
-    /// When text is inputed, the text is paging in the TextLabe size unit.
-    /// <returns>The total number of pages.</returns>
+    /// This is a class for stroing the text of a page.
     /// </summary>
     [EditorBrowsable(EditorBrowsableState.Never)]
-    public int SetText(TextLabel label, string str)
+    class PageData
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/Utility/TextPageUtil.cs:34]: this looks private class, but could you add private or internal or public explicitly?

File: src/Tizen.NUI/src/public/Utility/TextPageUtil.cs [hunk 2]
```diff
@@ -27,318 +27,331 @@
...
-        char[] charArray = new char[cnt];
-        pageString.CopyTo(data.startOffset, charArray, 0, cnt);
-        string pageText = data.previousTag+new String(charArray)+data.endTag;
-        return pageText;
-      }
-      return "";
-    }
-
-    private void SkipWhiteSpace(ref int offset)
+    class TagData
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/Utility/TextPageUtil.cs:46]: same as above

File: src/Tizen.NUI/src/public/Utility/TextPageUtil.cs [hunk 3]
```diff
@@ -27,318 +27,331 @@
...
     {
-      int character;
-      while( ( character = stream.Read()) != -1)
-      {
-        offset++;
-        if( character == WHITE_SPACE) continue;
-        else break;
-      }
+        public string tagName { get; set; }
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/Utility/TextPageUtil.cs:48]: is this really public? if so, please use pascal case~

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Review by AdunFang — APPROVED** [review]
It is OK.

**Pull request merged**